src/content/docs/logs/logs-context/configure-logs-context-php.mdx

```diff
@@ -215,7 +215,7 @@ If you don't see any logs for errors or traces, there may not be any for your ap
 
 ## Disable automatic logging [#disable-automatic]
 
-APM logs in context automatically forwards APM agent log data when enabled.  Currently this feature is disabled by default for the PHP agent.  However, in a future release it will be enabled by default. This can have a negative impact on your security, compliance, billing, or system performance. For more information, or if you need to adjust the default setting, follow the procedures to [disable automatic logging](/docs/logs/logs-context/disable-automatic-logging).
+APM logs in context automatically forwards APM agent log data when enabled.  As of agent release 10.3.0 this feature is enabled by default for the PHP agent.  This can have a negative impact on your security, compliance, billing, or system performance. For more information, or if you need to adjust the default setting, follow the procedures to [disable automatic logging](/docs/logs/logs-context/disable-automatic-logging).
```

Comment:
```suggestion
APM logs in context automatically forwards APM agent log data when enabled.  As of agent release 10.3.0, this feature is enabled by default for the PHP agent. This can have a negative impact on your security, compliance, billing, or system performance. For more information, or if you need to adjust the default setting, follow the procedures to [disable automatic logging](/docs/logs/logs-context/disable-automatic-logging).
```